core/templates/dev/head/components/help_modal_directive.html

```diff
@@ -0,0 +1,60 @@
+<script type="text/ng-template" id="modals/userHelp">
+<div class="modal-body oppia-long-text protractor-test-welcome-modal">
+  <h1>Need Help with Something?</h1>
+  <p>
+    You can check out the resources in our help center or
```

Comment:
Suggest: Check out the resources in our help center, or take a tour of the exploration editor.

(Note full stop at end.)